css/types/basic-shape.json

```diff
@@ -270,30 +270,46 @@
                 "partial_implementation": true,
                 "notes": "Only supported on the <code>offset-path</code> property."
               },
-              "firefox": {
-                "version_added": "63",
-                "partial_implementation": true,
-                "notes": "Only supported on the <code>clip-path</code> and <code>offset-path</code> properties.",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "layout.css.clip-path-path.enabled",
-                    "value_to_set": "true"
-                  }
-                ]
-              },
-              "firefox_android": {
-                "version_added": "63",
-                "partial_implementation": true,
-                "notes": "Only supported on the <code>clip-path</code> and <code>offset-path</code> properties.",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "layout.css.clip-path-path.enabled",
-                    "value_to_set": "true"
-                  }
-                ]
-              },
+              "firefox": [
+                {
+                  "version_added": "97",
+                  "partial_implementation": true,
+                  "notes": "It is supported on the <code>d</code> SVG presentation attribute, <code>clip-path</code> and <code>offset-path</code> properties."
```

Comment:
These notes (the originals included here) seem sketchy. `d` and `clip-path` are both SVG presentation attributes, right? And `offset-path` is a CSS property? Either we should make the distinction for each or for none, though I don't know which is right (since I'm a bit ignorant of SVG).

Reply:
Both offset-path and clip-path are CSS properties:

As I understand it,
- Generally CSS can only modify CSS properties. From v63 the `path()` method could be called on just 2 of these properties `clip-path` and `offset-path`
- "Generally" you can't set the value of SVG attributes using CSS - the set of SVG attributes that you can set using CSS are called [presentation attributes](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/Presentation#attr-d). `d` is one of these, so you should be able to set the value of it using `path()`. However you can't do that until FF97.
- Now there are other presentation attributes than `d`, but `d` is the only one that you can set with `path()` and path is what we are talking about here.

So from v63 the story was :

> "notes": "It is supported on <code>clip-path</code> and <code>offset-path</code> CSS properties."

The v97 and later story is:

> "notes": "It is supported on <code>clip-path</code> and <code>offset-path</code> CSS properties."
> It is supported on  the <code>d</code> SVG presentation attribute.

Hope that makes sense. 

I think you are suggesting I modify this as:
```suggestion
                  "notes": [
                    "It is supported on the <code>d</code> SVG presentation attribute",
                    "it is supported on the  <code>clip-path</code> and <code>offset-path</code> CSS properties."
                  ]
```